src/pretix/control/forms/__init__.py

```diff
@@ -300,18 +301,40 @@ def get_context(self, name, value, attrs):
 
 
 class MultipleLanguagesWidget(forms.CheckboxSelectMultiple):
+    template_name = 'pretixcontrol/multi_languages_select.html'
     option_template_name = 'pretixcontrol/multi_languages_widget.html'
 
     def sort(self):
-        self.choices = sorted(self.choices, key=lambda l: (
+        def filter_and_sort(choices, languages, cond=True):
+            return sorted(
+                [c for c in choices if (c[0] in languages) == cond],
+                key=lambda c: str(c[1])
+            )
+        self.choices = (
             (
-                0 if l[0] in settings.LANGUAGES_OFFICIAL
-                else (
-                    1 if l[0] not in settings.LANGUAGES_INCUBATING
-                    else 2
-                )
-            ), str(l[1])
-        ))
+                '',
+                filter_and_sort(self.choices, settings.LANGUAGES_OFFICIAL)
+            ),
+            (
+                (
+                    _('Community translations'),
+                    _('These translations are not maintained by the pretix team. We cannot vouch for their correctness '
+                        'and new or recently changed features might not be translated and will show in English instead. '
+                        'You can <a href="{translate_url}" target="_blank">help translating</a>.'.format(translate_url='https://translate.pretix.eu')),
```

Comment:
@wiffbi What's the purpose of the `.format` call? Should it make the URL not part of the translatable string? In that case it would need to be `).format()` instead of `.format())`

Reply:
Oh yes, good catch! Fixed with django’s `lazy_format`